<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmRegion</class>
 <widget class="QMainWindow" name="FrmRegion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestión de región </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>751</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Datos de la región    </string>
    </property>
    <widget class="QLabel" name="lbl_identificador">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Identificador</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_nombre">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_identificador">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_nombre">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_limpiar">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>40</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Limpiar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_agregar">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>80</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(87, 227, 137);</string>
     </property>
     <property name="text">
      <string>Agregar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_editar">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>120</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(153, 193, 241);</string>
     </property>
     <property name="text">
      <string>Editar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_eliminar">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>160</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(246, 97, 81);</string>
     </property>
     <property name="text">
      <string>Eliminar</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="le_buscar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>361</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_buscar">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QTableView" name="tbv_registrosRegion">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>751</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btn_limpiar</tabstop>
  <tabstop>btn_agregar</tabstop>
  <tabstop>btn_editar</tabstop>
  <tabstop>btn_eliminar</tabstop>
  <tabstop>le_identificador</tabstop>
  <tabstop>le_nombre</tabstop>
  <tabstop>le_buscar</tabstop>
  <tabstop>btn_buscar</tabstop>
  <tabstop>tbv_registrosRegion</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
